arm-servicebus/2015-08-01/eventhub.json

```diff
@@ -0,0 +1,1908 @@
+            "description": "The namespace name."
+          },
+          {
+            "name": "eventHubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The EventHub name."
+          },
+          {
+            "name": "consumerGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Consumer Group name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the consumer group description.",
+            "schema": {
+              "$ref": "#/definitions/ConsumerGroupResource"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventHub/namespaces/{namespaceName}/eventhubs/{eventHubName}/consumergroups": {
+      "get": {
+        "tags": [
+          "ConsumerGroup"
+        ],
+        "operationId": "ConsumerGroup_GetList",
+        "description": "Enumerates the consumer groups in a namespace. An empty feed is returned if no consumer group exists in the namespace.",
+        "externalDocs": {
+          "url": "https://msdn.microsoft.com/en-us/library/azure/mt639503.aspx"
+        },
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "namespaceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The namespace name."
+          },
+          {
+            "name": "eventHubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The EventHub name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved list of Consumer Group.",
+            "schema": {
+              "$ref": "#/definitions/ConsumerGroupListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CheckAvailabilityParameters": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the namespace"
+        },
+        "location": {
+          "type": "string",
+          "description": "Namespace location"
+        },
+        "tags": {
```

Comment:
Why would the service expect the user to provide tags while checking whether the namespace is available? Seems weird.